src/nupic/frameworks/opf/model.capnp

```diff
@@ -0,0 +1,9 @@
+@0xa2163632a7294a0d;
+
+using import "/nupic/frameworks/opf/opf_utils.capnp".InferenceType;
+
+# Next ID: 5
```

Comment:
@scottpurdy, I didn't notice this feedback before merging. It's fixed in my next task's changeset https://github.com/vitaly-krugl/nupic/tree/nup2463-serialize-predictedfield-and-inferenceenabled as part of NUP-2463